src/libraries/Microsoft.Internal.Runtime.AspNetCore.Transport/src/Microsoft.Internal.Runtime.AspNetCore.Transport.proj

```diff
@@ -13,6 +13,8 @@
     <PackageDescription>Internal transport package to provide aspnetcore with the assemblies from dotnet/runtime that make up the Microsoft.AspNetCore.App shared framework.</PackageDescription>
     <!-- Reference elements are missing from the nuspec: https://github.com/NuGet/Home/issues/8684. -->
     <NoWarn>$(NoWarn);NU5131</NoWarn>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>$(PatchVersion)</ServicingVersion>
```

Comment:
I think this should be harcoded to 1 as they will at somepoint get out of sync with the overall PatchVersion?

Reply:
These need to be shipped with every servicing release

Reply:
ah these are transport packages. I thought they were OOB packages. It makes sense now.